ModulBot/Cards/ActionButton.cs

```diff
@@ -0,0 +1,138 @@
+        {
+            int statId = Convert.ToInt32(buttonCallbackData[0]);
+            int actionType = Convert.ToInt32(buttonCallbackData[1]);
+
+            Bot.NextButtons = new List<List<InlineKeyboardButton>>();
+            if (actionType == (int)Constants.ActionTypes.ShowTurn)
+            {
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.TurnOn} {Constants.NotShowButtons}", Text = Constants.TurnOn } });
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.TurnOff} {Constants.NotShowButtons}", Text = Constants.TurnOff } });
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.Back} {Constants.NotShowButtons}", Text = Constants.Back } });
+            }
+            else
+            {
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.Subscribe} {Constants.NotShowButtons}", Text = Constants.Subscribe } });
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.Unsubscribe} {Constants.NotShowButtons}", Text = Constants.Unsubscribe } });
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.GetStat} {Constants.NotShowButtons}", Text = Constants.GetStat } });
+                Bot.NextButtons.Add(new List<InlineKeyboardButton> { new InlineKeyboardButton { CallbackData = $"{statId} {(int)Constants.ActionTypes.Back} {Constants.NotShowButtons}", Text = Constants.Back } });
+            }
+
+            var bd = new PostgresStatsRepository();
+            var stat = bd.GetById(statId);
+            await Bot.BotClient.EditMessageTextAsync(new ChatId(chatId), messageId, "Настройка статистики: " + stat.Name, replyMarkup: new InlineKeyboardMarkup(Bot.NextButtons));
+        }
+        private static string GetStat(int statId)
+        {
+            string _connStr = Startup.GetConnectionString();
+            int count = 0;
+            var db = new PostgresStatsRepository();
+            var selected_stat = db.GetById(statId);
+
+            try
+            {
+                using (var conn = new NpgsqlConnection(_connStr))
+                {
+                    conn.Open();
+                    string query = selected_stat.Query;
+                    var cmd = new NpgsqlCommand(query, conn);
+                    NpgsqlDataReader dr = cmd.ExecuteReader();
+                    while (dr.Read())
+                        count += 1;
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2228)